.circleci/config.yml

```diff
@@ -7,6 +7,9 @@ jobs:
       HOMEBREW_DEVELOPER: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
     steps:
+      - run:
+         name: Install unzip and svn for tests
+         command: sudo apt-get update && sudo apt-get install -y unzip subversion
```

Comment:
Not only the tests. The code assumes it too. Basically all the calls to `unzip` and `svn` in unpack_strategies.rb.

Reply:
wow, ok. We should then indeed ask for their advice on how to handle this on Linux without unzip/svn. Requirements maybe?

Reply:
> I think the unzip and svn problem was there before too. That code was in download_strategy.rb. So it was probably already broken. We just never noticed as there was no tests for that.

Yes, probably. Or vendoring. I think if the bug was there before, and nobody really complained, then it is probably not so urgent.
It will only bite those who build from source I guess.

Reply:
Wouldn't dependency collector add unzip or svn as a dependency on formula install?

Reply:
Right, actually it does ... so nothing to fix here.

It is just the tests that are broken, because the dependency collector does not kick in for that case.

Reply:
I was looking into a way to install from bottle just for the tests but I didn't get very far. When I finish it I'll probably send it upstream instead of here.

Reply:
You can do it, I tested it. But the test bot runs with --clean, which removes the bottle :D So this is not a good strategy.

Reply:
An alternative is to skip the test if /usr/bin/unzip or whatever isn't detected.